nativelink-store/tests/filesystem_store_test.rs

```diff
@@ -439,7 +449,7 @@ async fn file_continues_to_stream_on_content_replace_test() -> Result<(), Error>
     }
 
     // Replace content.
-    store.update_oneshot(digest1, VALUE2.into()).await?;
+    store.update_oneshot(digest1, VALUE1.into()).await?;
```

Comment:
Same sort of issue as with `digest_contents_replaced_continues_using_old_data`. I'm not totally sure this test should continue to exist.